packages/playback/src/lib/types/configuration.declarations.ts

```diff
@@ -61,10 +62,32 @@ export interface PlayerHlsConfiguration {
   customTagMap: CustomTagMap;
   transformTagValue: TransformTagValue;
   transformTagAttributes: TransformTagAttributes;
+
+}
+
+export interface PlayerEmeConfiguration {
+  /**
+   * Allows for using stored sessions. We store the session IDs internally.
+   * defaults to false
+   */
+  enablePersistentKeySessions: boolean;
```

Comment:
the naming might be confusing, I would name it something like:
`reusePersistentKeySessions`, with a description: the player will save persistent the mapping of the keyId to persisntSessionId to the localStorage and will attempt to load it if encounters the same keyId, and the session is still valid